### DISCUSSION TITLE
怎么才能突破128kb图片限制

### DISCUSSION THREAD

**Discussion by undefined-hestudio** [walinejs/waline#1769] [Q&A] [answered]
![Screenshot_2023-01-26-12-30-55-71_df198e732186825c8df26e3c5a10d7cd.jpg](https://user-images.githubusercontent.com/119711513/214758875-aed0f1f6-afb4-473d-9b11-0ef0f7d4b60e.jpg)

这个怎么突破这个限制？？

**Comment by lizheming** [comment, marked as answer]
https://github.com/walinejs/waline/discussions/1727
